components/org.wso2.carbon.micro.integrator.security/src/main/java/org/wso2/carbon/micro/integrator/security/vault/VaultTool.java

```diff
@@ -195,4 +210,24 @@ private static String doEncryption(Cipher cipher, String plainTextPwd) {
         System.out.println("\nEncryption is done Successfully\n");
         return encodedValue;
     }
+
+    private static void addSystemProperties() {
+        Properties properties = new Properties();
+        String filePath = System.getProperty(CONF_DIRECTORY_PATH) + File.separator + CARBON_PROPERTIES;
+        File file = new File(filePath);
+
+        if (file.exists()) {
+            try ( InputStream inputStream = new FileInputStream(file)) {
+                properties.load(inputStream);
```

Comment:
Do we need to load all properties from carbon.properties? Isn't org.wso2.SecureVaultPasswordRegEx sufficient?